### DISCUSSION TITLE
Mini upload speed

### DISCUSSION THREAD

**Discussion by lucky-zhao** [minio/minio#14500] [Q&A]
My Minio service takes about 1 minute to upload a 1m picture. How can I troubleshoot the problem? At present, it is known that I restart a docker container and upload quickly, but with the previous container, the upload is very slow。

My Minio is a distributed deployment, and the capacity used is about 2.5tb

**Comment by lucky-zhao** [comment]
The server itself has no problem and the bandwidth is normal. Because I restart a Minio container, the speed of uploading and downloading is very normal. It should be that the previous container has a problem

**Comment by lucky-zhao** [comment]
Excuse me, the number of small files stored in Minio is 2000000, with a total occupation of 2TB. Will this affect the upload speed

**Reply by meotimdihia** [reply]
My minio server has ~30 million images ~ 15TB and I don't see your problem.
